apps/bmd/src/components/ChoiceButton.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import styled from 'styled-components'
 
-import * as GLOBALS from '../config/globals'
+import { Button, Props as ButtonProps } from '@votingworks/ui'
```

Comment:
It looks like we inadvertently exported `Button`'s `Props` as just plain `Props`. That is surely not what we want. Let's change it to `ButtonProps` so this just becomes `import { Button, ButtonProps } from '@votingworks/ui'`

Reply:
Each component has plain ’Props’. 

Should we update each to be ’ComponentNameProps’?

I'm gonna assume so.

Reply:
Yeah, otherwise there's no way to e.g. `export * from './Button'` and get sensible names when importing `@votingworks/ui`. In fact, if they all have `Props` then I think the last one wins.